.buildkite/code.pipeline.yml

```diff
@@ -354,7 +354,7 @@ steps:
       - .buildkite/scripts/test_e2e.sh --timeout 20m --scenario e2e/runtime/runtime-encryption
     env:
       OASIS_TEE_HARDWARE: intel-sgx
-      OASIS_UNSAFE_MOCK_SGX: "1"
+      OASIS_UNSAFE_MOCK_TEE: "1"
```

Comment:
Should probably mention this in the internal changelog as this will require changes in other repositories (e.g. SDK).